runtime/bcutil/ConstantPoolMap.cpp

```diff
@@ -302,6 +302,14 @@ ConstantPoolMap::computeConstantPoolMapAndSizes()
 			}
 		}
 	}
+
+	J9ClassPatchMap *map = _context->patchMap();
```

Comment:
Would be good to comment here on why this is being done

Reply:
These are @fengxue-IS changes for Constant Pool Patching (#8460). These should go away after #8460 is merged, and this pull request is rebased. Only the four commits listed in the header description need to be reviewed for this pull request.